### PR TITLE
Initial manifest for CRUSH (2.50-1).

### PR DESCRIPTION
**Pull request by attipaci**
CRUSH is an astronomical data reduction and imaging pipeline (see [sigmyne.com/crush](https://www.sigmyne.com/crush)).

It runs on Java (>= 8).

I thought I might as well provide a flatpak release as well. This is the initial manifest for it for the current release (2.50-1).

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.sigmyne.crush

File: com.sigmyne.crush.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "com.sigmyne.crush",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk" ],
+    "command": "crush",
+    "finish-args": [
+   	"--share=network",
```
Comment (TingPing) [inline com.sigmyne.crush.json]: Whitespace problems throughout the file.

File: com.sigmyne.crush.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            "name": "crush",
+            "buildsystem": "simple",
+            "build-commands": [
+		"/usr/lib/sdk/openjdk/install.sh",
+    		"mkdir -p /app/share/crush",
+		"install -d Install/posix/share /app/",
+		"install -d Install/posix/man /app/share/",
+		"install -D -t /app/share/doc/crush Documentation/*",
+		"install -D -t /app/share/doc/crush copyright",
+		"install -D -t /app/share/doc/crush license.txt",
```
Comment (TingPing) [inline com.sigmyne.crush.json]: Set the mode for installed files.

**Comment by TingPing** [comment]
Needs appdata and ideally upstream involvement.

File: com.sigmyne.crush.json (hunk 1 above)
Comment (attipaci) [reply com.sigmyne.crush.json]: Thanks. I replaced leading white spaces with tabs. Hopefully that's what you meant...

File: com.sigmyne.crush.json (hunk 2 above)
Comment (attipaci) [reply com.sigmyne.crush.json]: Changed to using `cp -a` instead of `install` for moving files into location.

**Comment by attipaci** [comment]
I added an appdata (for now externally, but will be bundled into the next release -- in the few months). BTW, I am the sole upstream developer of CRUSH, so that part should be taken care of... :-)

**Comment by LeandroStanger** [comment]
bot, build com.sigmyne.crush

**Comment by LeandroStanger** [comment]
flathubbot: FAILURE
```
builddir/files/share/appdata/com.sigmyne.crush.appdata.xml: FAILED:
• tag-invalid           : <metadata_license> is not valid [GPL-3.0+]
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
Validation of files failed
```

**Comment by attipaci** [comment]
Made changes to appdata.xml to correct issues.

**Comment by LeandroStanger** [comment]
bot, build com.sigmyne.crush

**Comment by LeandroStanger** [comment]
bot, build com.sigmyne.crush

**Comment by LeandroStanger** [comment]
flathubbot: WARNINGS
```
builddir/files/share/appdata/com.sigmyne.crush.appdata.xml: FAILED:
• tag-missing           : <translation> not specified
• tag-invalid           : XML data contains unknown tag
• aspect-ratio-invalid  : <screenshot> aspect ratio not 16:9 [https://www.sigmyne.com/crush/screenshots/crush-screenshot1.jpg]
• aspect-ratio-invalid  : <screenshot> aspect ratio not 16:9 [https://www.sigmyne.com/crush/screenshots/crush-screenshot4.jpg]
• translations-required  : <name> has no translations
• translations-required  : <summary> has no translations
• translations-required  : <description> has no translations
Validation of files failed
```

**Comment by attipaci** [comment]
Fixes to appdata.xml warnings (New screenshot with 16:9 aspect ratio, added translation tag). Not sure what is the unknown tag it complains about. `appstream util validate` does not report anything wrong...

**Comment by LeandroStanger** [comment]
bot, build com.sigmyne.crush

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.sigmyne.crush

**Pull request closed without merging**